app/views/shared/_header_categories.html.erb

```diff
@@ -0,0 +1,28 @@
+<nav class='header-container'>
+  <ul>
+    <li>
+      <%= link_to user_path(@user) do %>
+        <%= image_tag image_path('hamburger-menu.jpg'), style: 'width: 4rem; height: 4rem;' %>
+      <% end %>
+    </li>
+    <li class="nav-item">
+      CATEGORIES
+    </li>
+      <% if user_signed_in? %>
+        <li class="nav-item">
+          <%= button_to destroy_user_session_path, method: :delete do %>
+            <%= image_tag 'search.png', alt: 'Sign out', style: 'width: 4rem; height: 4rem;' %>
+          <% end %>
+        </li>
+        <% else %>
+        <li class="nav-item my-flex">
+          <%= link_to "Login", new_user_session_path, class:"button" %>
+          <%= link_to "Signup", new_user_registration_path, class:"button" %>
+        </li>
+      <% end %>
+  </ul>
+</nav>
+<div class="gift-credit-container details-heading">
+  <h2>Gift credit</h2>
+  <h2>$300</h2>
```

Comment:
- [ ] [REQUIRED] In as much as I understand your explanation, I will suggest that you remove the gift credit that appears upon a user's Login to not confuse the user just as stated by the previous reviewer. This is not part of the project requirement. Let's leave the entire Category page to the control of the current logged-in user.


<img width="379" alt="Screenshot 2023-12-02 at 19 07 40" src="https://github.com/diarisdiakite/my-account/assets/58692590/902e2693-8656-45d1-89dd-12967b936bfd">